src/metabase/pulse/render/body.clj

```diff
@@ -193,15 +203,18 @@
   [_ _ timezone-id :- (s/maybe s/Str) card {:keys [cols] :as data}]
   (let [[x-axis-rowfn y-axis-rowfn] (common/graphing-column-row-fns card data)
         rows                        (common/non-nil-rows x-axis-rowfn y-axis-rowfn (:rows data))
-        max-value                   (apply max (map y-axis-rowfn rows))]
+        row-values                  (map y-axis-rowfn rows)
+        min-value                   (min 0 (apply min row-values))
+        max-value                   (apply max row-values)]
     {:attachments
      nil
 
      :content
      [:div
       (table/render-table (color/make-color-selector data (:visualization_settings card))
+                          (normalize-bar-value 0 min-value max-value)
                           (mapv :name cols)
-                          (prep-for-html-rendering timezone-id card data y-axis-rowfn max-value 2))
+                          (prep-for-html-rendering timezone-id card data y-axis-rowfn min-value max-value 2))
```

Comment:
Same feedback as the comment about `query-results->row-seq` -- 7 positional args is pretty hairy, I'd strongly consider reworking the function to take named arguments of some sort